client/my-sites/plans/navigation.js

```diff
@@ -89,39 +93,43 @@ class PlansNavigation extends React.Component {
 				onMobileNavPanelOpen={ this.onMobileNavPanelOpen }
 			>
 				<NavTabs label="Section" selectedText={ sectionTitle }>
-					{ hasPlan && (
-						<NavItem
-							path={ `/plans/my-plan/${ site.slug }` }
-							key="myPlan"
-							selected={ path === '/plans/my-plan' }
-						>
-							{ translate( 'My Plan' ) }
-						</NavItem>
-					) }
-					<NavItem
-						path={ `/plans/${ site.slug }` }
-						key="plans"
-						selected={ path === '/plans' || path === '/plans/monthly' }
-					>
-						{ translate( 'Plans' ) }
-					</NavItem>
-					{ canManageDomain && (
-						<NavItem
-							path={ `/domains/manage/${ site.slug }` }
-							key="domains"
-							selected={ path === '/domains/manage' || path === '/domains/add' }
-						>
-							{ translate( 'Domains' ) }
-						</NavItem>
-					) }
-					{ canManageDomain && (
-						<NavItem
-							path={ `/domains/manage/email/${ site.slug }` }
-							key="googleApps"
-							selected={ path === '/domains/manage/email' }
-						>
-							{ translate( 'Email' ) }
-						</NavItem>
+					{ site && (
```

Comment:
This section relies on `site.slug` or we'd get nasty routes or errors. Conditionally render the contents.